src/language/macros.ts

```diff
@@ -0,0 +1,100 @@
+import { MacroType } from '@grafana/experimental';
+
+const COLUMN = 'column',
+  RELATIVE_TIME_STRING = "'1m'";
+
+export const MACROS = [
+  {
+    id: '$__timeFilter(dateColumn)',
+    name: '$__timeFilter(dateColumn)',
+    text: '$__timeFilter',
+    args: [COLUMN],
+    type: MacroType.Filter,
+    description:
+      "Will be replaced by a time range filter using the specified column name. For example, time BETWEEN '2017-07-18T11:15:52Z' AND '2017-07-18T11:15:52Z'",
+  },
```

Comment:
it would be nice to merge this with the macros defined in `Suggestions.ts` so you don't need to define things in two different places

Reply:
Hoping to be able to remove Suggestions.ts very soon! It's the last step of [this epic](https://github.com/grafana/redshift-datasource/issues/108).